src/scripts/Dockerfile.android

```diff
@@ -0,0 +1,22 @@
+FROM debian:stretch AS debian
```

Comment:
What about deriving from `bitriseio/android-ndk` directly? If there is anyone in the industry I'd trust for maintaining an Android build image, it is the bitrise team

Reply:
I have no opinion on this. @devnexen wdyt?

Reply:
I ve done that in another project and thought it was a bit ancient but that s a fair point maybe that would just work in botan's case.